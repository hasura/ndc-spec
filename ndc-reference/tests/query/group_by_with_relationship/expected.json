[
  {
    "rows": [
      {
        "first_name": "Peter",
        "last_name": "Landin",
        "articles": {
          "groups": [
            {
              "dimensions": {
                "author": "The Next 700 Programming Languages"
              },
              "aggregates": {
                "min_id": 1,
                "max_id": 1
              }
            }
          ]
        }
      },
      {
        "first_name": "John",
        "last_name": "Hughes",
        "articles": {
          "groups": [
            {
              "dimensions": {
                "author": "Why Functional Programming Matters"
              },
              "aggregates": {
                "min_id": 2,
                "max_id": 2
              }
            },
            {
              "dimensions": {
                "author": "The Design And Implementation Of Programming Languages"
              },
              "aggregates": {
                "min_id": 3,
                "max_id": 3
              }
            }
          ]
        }
      }
    ]
  }
]