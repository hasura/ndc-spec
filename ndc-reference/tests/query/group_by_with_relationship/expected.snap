---
source: ndc-reference/bin/reference/main.rs
expression: response.0
input_file: ndc-reference/tests/query/group_by_with_relationship/request.json
---
[
  {
    "rows": [
      {
        "first_name": "Peter",
        "last_name": "Landin",
        "articles": {
          "groups": [
            {
              "dimensions": [
                "The Next 700 Programming Languages"
              ],
              "aggregates": {
                "min_id": 1,
                "max_id": 1
              }
            }
          ]
        }
      },
      {
        "first_name": "John",
        "last_name": "Hughes",
        "articles": {
          "groups": [
            {
              "dimensions": [
                "Why Functional Programming Matters"
              ],
              "aggregates": {
                "min_id": 2,
                "max_id": 2
              }
            },
            {
              "dimensions": [
                "The Design And Implementation Of Programming Languages"
              ],
              "aggregates": {
                "min_id": 3,
                "max_id": 3
              }
            }
          ]
        }
      }
    ]
  }
]
